### DISCUSSION TITLE
Space stations and potential ways of using them to improve the feeling of Human space

### DISCUSSION THREAD

**Discussion by bene-dictator** [endless-sky/endless-sky#8785] [General]
While planets carry a lot of the narrative and cultural weight that each region of human space has, the lack of human space infrastructure leads to some holes that I feel like could be filled and improved upon to make each region stand out more. There has been talk of creating more stations (from Anarchist, Amazinite and Azure) but nothing concrete yet so this discussion could be used to gather our thoughts into one place.

--Thoughts
Human space has quite a few space stations, but they are not as widespread or developed as I had expected for a spacefaring species with 900 years of experience. The Deep's space stations are the only ones that show their development; for a area so small, they have twice as many stations as the Rim and as many as the whole of the Core, which is a huge populated area. Muninn and Huginn's existence is great for showing the player the Deep's isolation and might, while the Deep's other two stations that produce deuterium solely for the Deep is also gets across the idea of the region's self-reliance and independence. The lack of research stations apart from one of the deuterium plants is strange for the Deep's scientific focus (although maybe Alexandria takes that spot, but a station just about research and not archival would be better)

However, I think that the Core could easily have more space stations as it is relatively more well settled than the Deep, because despite their superiority Earth is still the centre of human space and the Core is much closer and populated, and it would make sense that some of that population would be offloaded into research, living, or factory stations. 
- Sundive's observatory might serve better as a observatory on a space station.
- Millrace would have orbital habitats due to overcrowding and Syndicate executives wanting to spend their allotted budget before deadlines. 
For that Millrace one, I don't think some stations need to make that much sense as long as they serve well narratively and bring across a point about the region's culture better.

The Paradise Planets are called the Paradise Planets, but I see very little of that wealth being spent in space where it is more visible to the player. For all the eccentricity that we could see in characters like David Turner I'm surprised Mountaintop is the first space station he's had a role in building – and also I'm surprised that Typhon Station, which is really in the Far North, is the only station that could have been developed by the Paradise Planets. This could be used to show multiple things about the Paradise Planets and the Southern Rim:
- For the Paradise Planets' xenophobic attitude I wonder why they don't have more military stations against the pirates.
- And for all the wealth sloshing around I wonder why there aren't any gambling stations either.
- Luna's status as an underdeveloped but sprawling shipyard could be better put into perspective if Gemini Shipyards was a space station, where it's more practical and efficient to build ships, and could bring across the idea that the Paradise Planets are taking money from the South and the Near Earth planets and funneling them onto grand or frivolous projects that don't benefit those citizens paying for them.
- Political tensions between South and North could be better expressed if the South were banned from building new stations while the North continued to build them, with the FW's independence leading an uptick in building stations for the new FW military. I could see a station serving the gap in Alpha Arae, for example.

The timeline mentions constant warfare between space corporate states but there is very little to show in terms of stations. For example, bringing a student studying the Turf War to an abandoned station in the Dirt Belt to study its history and how it delineated the front lines of various corporate states, then the Alphas, then the Republic, until it was forgotten. Very little of the Republic's history is really shown in game and the state of some stations (not just Hydra Station) could be used to show the Republic's decadence.

**Comment by Quantumshark** [comment]
Sundive - yes, absolutely this would make more sense as a space station (or alternatively, a settlement on a planet or moon with no atmosphere). Atmosphere makes observing things harder, so you'd absolutely want to do that from space rather than from the ground. Probably this would be a space station at an L2 point, so that there's no interference from the sun either.

Millrace - I'm less convinced about this one - even with ES's repulsor tech, it's still going to be more expensive to put things in orbit than to put them on a planet, and needing to enter and leave orbit to commute to work would represent a significant inefficiency that would cut into profits.

Geminus - this would absolutely make sense, for all the reasons you mention.

Paradise Planets - Gambling stations and other similar luxury things would absolutely make sense.
Military ones possibly less so - I can imagine the wealthy would see them as an eyesore, and likely would end up with the "yes, we need more of these... but not here." school of thought, so the military stations would end up in systems along the edges of the paradise worlds rather than interspersed among them.

South station ban - I don't think saying the construction of stations was outright banned in the south really makes sense - it'd fit better with existing content if you instead said there was a lack of stations because Parliament refused to allocate construction funding to the South, rather than because they banned the construction of stations there.

**Reply by bene-dictator** [reply]
> Millrace - I'm less convinced about this one - even with ES's repulsor tech, it's still going to be more expensive to put things in orbit than to put them on a planet, and needing to enter and leave orbit to commute to work would represent a significant inefficiency that would cut into profits.
Fair enough. However I still think that if it can serve a point, it would work. Building the station just so budget money isn't taken from an executive, for example, is silly, but it could show how executives are more worried about themselves than the company or the money they're dealing with, for example.

> South station ban - I don't think saying the construction of stations was outright banned in the south really makes sense - it'd fit better with existing content if you instead said there was a lack of stations because Parliament refused to allocate construction funding to the South, rather than because they banned the construction of stations there.
Yeah, that makes far more sense.

**Comment by TheMarksman-ES** [comment]
Another thing to note is that before the prevalance of the Anti Gravity Repulsors, there would've been much higher demands for orbital infrastructure to support SSTO, space based construction, and other space related infrastucutre that would've cost a fortune to launch to space. 
Then again, once getting to space became a lot easier, space based infrasture would've been less in demand.

**Comment by lumbar527** [comment]
> The Paradise Planets are called the Paradise Planets, but I see very little of that wealth being spent in space where it is more visible to the player.

I think the Paradise planets are spending their money on terraforming and maybe weather.

> - For the Paradise Planets' xenophobic attitude I wonder why they don't have more military stations against the pirates.

I agree with this:
> Military ones possibly less so - I can imagine the wealthy would see them as an eyesore, and likely would end up with the "yes, we need more of these... but not here." school of thought, so the military stations would end up in systems along the edges of the paradise worlds rather than interspersed among them.

Also, the Paradise planets think of the Navy as their personal defense force, and I think that's because of the large taxes or extra money they pay. Then they wouldn't need as many extra defenses.

**Comment by MasterOfGrey** [comment]
> Human space has quite a few space stations, but they are not as widespread or developed as I had expected for a spacefaring species with 900 years of experience. 

Ultimately, this is probably related to one main thing: Stations need art assets made, and that takes time and artists who are motivated by the idea of doing so.

I feel fairly certain from this discussion though, that if an artist makes a station, we can find a use for it.

**Comment by yafflehk** [comment]
Military stations with long range weapons would be fun.

**Reply by mOctave** [reply]
Yes, although that would also have some minor mechanics issues that would need sorting out before it would work.

Also, if we armed stations, we would probably want to arm planets as well, and that could get confusing and throw a wrench in the combat mechanics (although planets/stations with heavy anti-ship weapons could be a way to balance ship size, I suppose)

As a sidenote:
I had a similar idea for this kind of thing for the Searchers, and plan to have a giant, one-of-a-kind, uncapturable superheavy that is basically an armed station/carrier, playing a role in the plot and potentially deciding the outcome of a major battle.

**Comment by Tangle10** [comment]
Apologies if this _is_ actually considered a necro here, but I have some thoughts on this:

1) As it stands, it seems like most of the stations in Human space are fuelling stations constructed by the Deep during the Caravan Era. I think this means that each station, including the extant ones, provides a unique opportunity for historical exposition- Charon Station/Aldhibain mentions it was the first place south of Hydra Station/Algorel that caravans could stop to refuel, implying the nonexistence or lack of fuel available at settlements in the five systems between them; Wyvern Station/Izar is called out as an end point for merchant routes, synergizing with Starcross/Acrux and implying the modern Typhon Station/Hadar did not exist at the time (since it's just a jump away). Elaborating on which of these stations are modern, which are four centuries old, and which are older (and who built them!) dictates the shape of Human space at the time, which while not necessary relevant to First Last and the modern "refuel on any planet" setup can bring about quite a few interesting stories- for instance, HistoricalTimeline notes that the Deep caravan system directly led to the rise of the Syndicate and implies that this also helped the Republic rise.

2) While the bottleneck of "we'd like to not overuse each station sprite" is not to be overlooked, there's certainly plenty of room for more. Assuming refueling in that era is only possible at stations or very highly populated places, there's one major place a refueling station would likely be desired: the Eastern Dirt Belt, While most of the Dirt Belt was uninhabited at the time, and Tarazed _can_ be reached in six jumps from Aldhibain, that's a circuitous sixteen-jump route from Sol- and even longer from the then-developing Syndicate or the populated worlds of the Deep. Running through the Eastern Dirt Belt instead takes nine jumps from Sol or the groundside refueling station at Shiver/Altair, which is probably longer than even the merchant caravans could manage without relying on ramscoops- making them prime pirate targets, incidentally. Cebalrai is the most convenient spot for a refueling station- while Ascella is closer to Tarazed (five jumps to Sol and four to Tarazed rather than Cebalrai's 3 to Sol and 6 to Tarazed), Cebalrai is considerably less out-of-the-way for other journeys- four and six jumps to the preexisting refueling stations at Hydra Station/Algorel and Charon Station/Aldhibain respectively, making it not only a useful gateway between the center of Human space and Quarg space but also the entire South and West with the center and East. There are a couple other candidates- Lesath would, in concert with Cebalrai, provide a gateway from the Sol and Tarazed regions to Grendel Station/Men (now Smuggler's Den): stated to be among the longest-inhabited regions in Human space; and there's certainly room to consider a station in the Far North for "going over the top"- and I think it would do well to add flavor by adding nonlandable stations (like in Korath space) as well.

3) Of course, I'm obsessed with refueling stations and that's far from the only use for space stations. While a lot of space station goals are somewhat obviated by Endless Sky's ubiquitous repulsors, there are still some places aside from those mentioned further up for stations to fit- in particular, the Turf Wars between corporations for mining rights in the Dirt Belt imply that space mining was perhaps even more common than it is now. As it took three hundred years for more of the Dirt Belt to host colonies, it'd make sense for space stations to serve as bases for mining instead. Systems like Ascella on one end and Orbona on the other are rich in lucrative space resources, and especially in the 2400s would have a desperate need for a place for ships to land and offload. On the other hand, if indeed the mining is supposed to have taken place on the planets of the Dirt Belt... it'd be nice to have more history available.

wow this is long. hope you enjoy

EDIT 1: Of course, there's a problem with putting a station in Cebalrai: There's no gas giant in that system. With that rather important factor in mind, Rasalhague is the best option: 4 jumps from Sol and Altair, 5 jumps to Algorel and Tarazed, and 6 jumps (by the Alnasl route) to Aldhibain. Lesath also lacks a gas giant, meaning for the Men-Tarazed route appropriate midpoints are Rastaban and Girtab. Probably Rastaban, as the Men-Tarazed station is probably only useful if the "nexus" station exists as well (there's considerably less reason to have a station in Rastaban if it doesn't provide an alternative route to Sol from the [Men/Tarazed]-Aldhibain-Algorel-Sol route). If there were a station in the Far North, I'd probably put it in Sumar if possible- 4 jumps to Triton Station/Alheka, Carbuncle Station/Elnath, and Oberon Station/Menkar; though we don't know how many of those existed at the time. It might also be convenient as a pirate hideout, 5 jumps from Freedom/Almaaz and in the modern era 4 jumps from Stormhold/Alcyone...

**Reply by pepoluan** [reply]
> While the bottleneck of "we'd like to not overuse each station sprite" is not to be overlooked

What's wrong with building space stations according to blueprints? That will very much lead to similarity between space stations.

After 9 centuries, economy of scale should be well-known... and that includes building spacestations from same part, according to same plan, etc. Differing only on attachments.

**Reply by Tangle10** [reply]
I'd agree, @pepouluan, but there's the problem that players might well get confused if the same station sprite is used a lot. I'm not sure I agree with that being much of a problem, but it's an understandable concer.

**Comment by bene-dictator** [comment]
<img width="447" alt="Screenshot 2023-11-26 at 6 13 06 PM" src="https://github.com/endless-sky/endless-sky/assets/115441627/16991f37-fe4a-4c86-b0ac-fac98fbf5119">
<img width="1440" alt="Screenshot 2023-11-26 at 6 14 03 PM" src="https://github.com/endless-sky/endless-sky/assets/115441627/3eaaf256-175f-4e0f-a662-9bab60fee9a4">
<img width="1440" alt="Screenshot 2023-11-26 at 6 15 25 PM" src="https://github.com/endless-sky/endless-sky/assets/115441627/d2f10f03-a9e6-4ae3-894b-6a50494c87f1">
<img width="524" alt="Screenshot 2023-11-26 at 6 20 13 PM" src="https://github.com/endless-sky/endless-sky/assets/115441627/9ce293ab-47fd-4b14-9cd6-84ef67dad37f">
<img width="1440" alt="Screenshot 2023-11-26 at 6 20 51 PM" src="https://github.com/endless-sky/endless-sky/assets/115441627/39b49e08-d7e7-4a0a-9ffa-9617ff0dbfdc">

I'm trying to make station sprites, but I'm no good at it. ~~also my inconsistent US/GB spelling.~~

**Reply by Tangle10** [reply]
Interesting, though! Both quite cute concepts.

**Reply by mOctave** [reply]
They look good to me!
